sectionsUtilityTool/build.sh

```diff
@@ -1,23 +1,19 @@
 #!/bin/sh --
```

Comment:
It would be good to add a comment about where to find the build script in the top level readme in the canvas project.